src/App.tsx

```diff
@@ -76,35 +81,39 @@ function App() {
   );
 
   useEffect(() => {
-    async function getAttributes(dataElementURI: string) {
+    async function getAttributes(NBClass: string, dataElementURI: string) {
```

Comment:
How about `NBResource` instead of `Class`. Class is a bit generic for me

```suggestion
    async function getAttributes(NBResource: string, dataElementURI: string) {
```